packages/core/src/core.ts

```diff
@@ -166,4 +183,23 @@ export class Core extends ICore {
       throw error;
     }
   }
+
+  private dispatchEnvelope = ({ url }: { url: string }) => {
+    if (!url || !url.includes("wc_ev") || !url.includes("topic")) return;
+
+    const getSearchParamFromURL = (_url: string, param: any) => {
```

Comment:
@ganchoradkov should we move this function to utils? where?

Reply:
`getSearchParamFromURL` for sure